prep-example/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>prep-example</title>
+</head>
+<body>
+  <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+
+
+
+  </p>
+</body>
+</html>
+With my new skills, I would like to build an app that could scan and solve math problems!
```

Comment:
Hi Paul, try moving this text to line 10

Reply:
> Hi Paul, try moving this text to line 10

Hey tim, im not quiet sure how to do that. Can you help me?

Reply:
Sure thing, the feedback guide had steps you can follow. If you have questions please send me a slack message

https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/pull-request-feedback